navbar.css

```diff
@@ -0,0 +1,40 @@
+#navbar{
+    height: 50px;
+    display: grid;
+    grid-template-columns: 100px 100px;
+    grid-gap: 10px;
+    position: fixed;
+    top:0px;
+    left: 0px;
+    right: 0px;
+    z-index: 100;
+    background-color: rgb(138, 46, 46);
+    
```

Comment:
The extra space should be removed.

Reply:
> The extra space should be removed.

- [OPTIONAL] Just as the previous reviewer pointed, every comment should be given with reason.